src/main/java/seedu/address/model/order/DeliveryDateTime.java

```diff
@@ -13,10 +13,11 @@
  */
 public class DeliveryDateTime {
 
-    public static final String MESSAGE_CONSTRAINTS = "DeliveryDateTime should be in the format dd/MM/yyyy HH:mm "
-            + "and should be a valid date and time before today's date!";
+    public static final String MESSAGE_CONSTRAINTS = "DeliveryDateTime should be in the format dd-MM-yyyy HH:mm "
+            + "or a natural date (e.g. Mon 22:30, Wednesday 10:20) "
+            + "and should be a valid date and time after today's date!";
```

Comment:
are we allowing the addition of DeliveryDateTime from the past in this PR or another?

Reply:
Let's do it in another PR. I'll change the comment.

Reply:
Hold on sorry, I thought this was a grammatical error (see previous file version) so I change it to match with the current implementation where orders before input datetime is rejected. What do you think? Or is this not the issue you are mentioning?